server/models/Donation.js

```diff
@@ -1,4 +1,5 @@
 const type = require('../constants/transactions').type.DONATION;
+const donationLib = require('../lib/donation');
```

Comment:
should it be `donations.js`? We seem to use both plural and singular in libs; controllers are all plural.